ceph-dashboard-pull-requests/config/definitions/ceph-dashboard-pull-requests.yml

```diff
@@ -68,6 +68,11 @@
             - ../../../scripts/dashboard/install-e2e-test-deps.sh
       - shell: |
           export CYPRESS_ARGS="--record --key $CYPRESS_RECORD_KEY --tag $ghprbTargetBranch" COMMIT_INFO_MESSAGE="$ghprbPullTitle"
+          export APPLITOOLS_BATCH_ID="PR-${ghprbPullId}_${BUILD_TAG}"
```

Comment:
I don't know how the PR test is green but these aren't double bracketed.  Maybe JJB only tried to template the {JOB_NAME} with " in front of it?  Can you {{ }} these too please?

Reply:
I guess because that's not a template job (while the nightly job is), could be?

Reply:
Oh, yes of course.  K, LGTM.